cli/cli_connect_test.go

```diff
@@ -18,39 +20,92 @@ import (
 	"google.golang.org/grpc"
 )
 
+func captureOutput(f func()) (string, error) {
+	reader, writer, err := os.Pipe()
+	if err != nil {
+		return "", err
+	}
+	stdout := os.Stdout
+	stderr := os.Stderr
+	defer func() {
+		os.Stdout = stdout
+		os.Stderr = stderr
+	}()
+	os.Stdout = writer
+	os.Stderr = writer
+	out := make(chan string)
+	wg := new(sync.WaitGroup)
+	wg.Add(1)
+	go func() {
+		var buf bytes.Buffer
+		wg.Done()
+		io.Copy(&buf, reader)
```

Comment:
At first I wanted to ask why is pipe used if stdout can be written straight to buffer. Then I realized that `Copy` is used specifically because it only exits on EOF, otherwise we might not capture the full output, and Pipe is needed because Copy requires a Reader argument. So would be nice to leave a comment explaining this I think :D